README.md

```diff
@@ -43,6 +43,18 @@ On Linux and Mac `dev-env` can be installed with:
 1. Install Nix by running: `bash <(curl https://nixos.org/nix/install)`
 2. Enter `dev-env` by running: `eval "$(dev-env/bin/dade assist)"`
 
+> Note: For **macOS Catalina** users, there are some extra steps required to
+> get `nix` working, as it requires access to a top-level `/nix` directory. The
+> following steps will create a suitable `/nix` folder.
+> 1. Run `sudo bash -c "echo nix >> /etc/synthetic.conf"`
```

Comment:
Perhaps use `sudo tee` instead of embedded shell?

```suggestion
> 1. Run `echo nix | sudo tee -a /etc/synthetic.conf`
```

Reply:
Why?

Reply:
Because (a) the user's default shell probably won't be `bash` (it's `zsh` by default on Catalina), and (b) you're invoking a parser, which always gives me the heebie jeebies. This is not at all important, though; feel free to ignore it if you disagree.

Reply:
What does the default shell have to do with this? I'm explicitly invoking `bash`. (My default shell is `zsh`.)

The only way this wouldn't work is if you're using a shell that will not give the string to bash in one piece, i.e. you are using a shell that doesn't parse double quotes in the correct (i.e. POSIX) way. If you're in that situation I'm likely not qualified to help and hopefully you know why and how to work with it.